content.js

```diff
@@ -167,39 +167,52 @@ if (window.contentScriptInjected !== true) {
 
     function looksLikeForeignWord(word) {
         word = word.trim().toLowerCase();
+        if (word == "")
+            return false;
+        
+        const serbian_words_with_double_letters = [
+            "poddres",
+            "vakuum",
+            "kontinuum",
+            "zoo",
+            "vodootpor",
+            "tihookeansk"
+        ];
+        
+        for (var serbian_word of serbian_words_with_double_letters) {
```

Comment:
This for loop is the same as for loop on line `212`. To avoid code duplication write a separate function which has parameters the word you are inspecting and array of words you're looking for, then you can use it in both cases and at the same time simplify this function.